http/README.md

```diff
@@ -0,0 +1,133 @@
+# HTTP
+
+| Lifecycle Stage | Maturity      | Status | Latest Revision |
+| --------------- | ------------- | ------ | --------------- |
+| 1A              | Working Draft | Active | r0, 2023-01-23  |
+
+Authors: [@marten-seemann], [@MarcoPolo]
+
+Interest Group: [@lidel], [@thomaseizinger]
+
+[@marten-seemann]: https://github.com/marten-seemann
+[@MarcoPolo]: https://github.com/MarcoPolo
+[@lidel]: https://github.com/lidel
+[@thomaseizinger]: https://github.com/thomaseizinger
+
+## Introduction
+
+This document defines how libp2p nodes can offer and use an HTTP transport alongside their other transports to support application protocols with HTTP semantics. This allows a wider variety of nodes to participate in the libp2p network, for example:
+
+- Browsers communicating with other libp2p nodes without needing a WebSocket, WebTransport, or WebRTC connection.
+- HTTP only edge workers can run application protocols and respond to peers on the network.
+- `curl` from the command line can make requests to other libp2p nodes.
+
+The HTTP transport will also allow application protocols to make use of HTTP intermediaries such as HTTP caching, and layer 7 proxying and load balancing. This is all in addition to the existing features that libp2p provides such as:
```

Comment:
Related to https://github.com/libp2p/specs/pull/508/files#r1541421460 but not all HTTP methods are equally cachable/proxyable - should the spec have an opinion on how requests are sent?

Reply:
On balance, it'll probably be application specific.

Reply:
Applications will interact with HTTP semantics, so they will have directly control how requests are sent.

Applications may choose to use helper libraries that can map some other request/response protocol onto HTTP semantics. In that case, that library defines the mapping. This spec itself only suggests one way of doing that with `POST`, but other ways are also valid.